.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -6,6 +6,8 @@ labels: 'Type: enhancement'
 assignees: mastercoms
 
 ---
+**NOTE:Make sure the feature being requested does NOT change the gameplay balance in a significant way and is NOT purely a cosmetic choice**
+This poject is not interested in re-balancing the game or changing its looks. Ex. "Buff the Bison", "Bring back the beta muzzle flashes" would not be accepted
```

Comment:
project

Reply:
I think it's either "its look" or "it looks" not sure tho

Reply:
"its looks" is valid, but might change that if its sound weird

Reply:
Ah, you meant plural.

Reply:
yup, but if you looked at it and it was weird, its worth changing.
also fixed the typo

Reply:
I just didn't think of the plural. I thought you meant look as in one big thing.